README.md

```diff
@@ -7,7 +7,18 @@
     <li>Book purchasing: Users have the option to purchase books directly through this platform, conveniently selecting desired books and putting them into a shopping cart.</li>
 </ul>
 <h1 style="font-size: 24.5px; margin-top: -1pc; margin-left: 0pc;">Technology stack</h1>
-<p style="font-size: 16px; margin-top: -0.8pc;">While working on the project, I used the following technologies: <span style="color: black; font-weight: bold;">Spring Boot, Spring Security, Spring Data JPA, Spring MVC, Swagger, Java, Hibernate, MySQL, Spring, Maven, JUnit, Mockito, Windows, Servlets, Tomcat, JSP, XML, JSON, Git, IDE.</span></p>
+<p style="font-size: 16px; margin-top: -0.8pc;">While working on the project, I used the following technologies for the following purposes:</p>
+<ul style="font-size: 16px; padding-left: 20px; margin-top: -0.8pc; margin-left: 0pc;">

```

Comment:
I would also suggest to learn how to use Markdown syntax instead of HTML. Markdown is much more convenient and has styles predefined by GitHub.

Reply:
I regarded that, but somehow it doesn't work properly on my laptop.

Reply:
Strange, because it is not a 'laptop' specific feature, it's .MD format.